workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSComponent.cs

```diff
@@ -11,5 +11,29 @@ public class SpatialOSComponent : MonoBehaviour
         public Entity Entity;
         public World World;
         public WorkerSystem Worker;
+
+        private EntityManager entityManager;
+
+        public bool TryGetGameObjectForEntity(EntityId entityId, out GameObject linkedGameObject)
```

Comment:
I think`TryGetGameObjectFromSpatialOSEntityId` I think is what the rest of the convetion is. We can probably shorten that but we need to be clear it's not an ECS entity.